src/sentry/lang/javascript/processor.py

```diff
@@ -498,7 +498,11 @@ def is_utf8(encoding):
     if encoding is None:
         return True
     try:
-        return codecs.lookup(encoding).name == 'utf-8'
+        return codecs.lookup(encoding).name in (
+            'utf-8',
+            # latin-1 should be just a subset of utf-8, so is safe
```

Comment:
this comment makes no sense. latin-1 is not a subset of utf-8, it's a 8bit encoding that always succeeds where the first 256 characters directly map to unicode codepoints.